test/models/ApprovalStatusSpec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import org.scalatest.{MustMatchers, OptionValues, WordSpec}
+import play.api.libs.json.{JsError, Json, JsString, JsObject, JsNull}
+import models.ApprovalStatus._
+
+class ApprovalStatusSpec extends WordSpec with MustMatchers with OptionValues with Enumerable.Implicits {
+
+  "CompanyApplyingIsRightsHolder" must {
```

Comment:
Minor: the description of the "must" doesn't tie in to the test

Reply:
ha yes, will update